flytekit/contrib/notebook.py

```diff
@@ -0,0 +1,558 @@
+from __future__ import absolute_import
+
+from __future__ import print_function
+import os as _os
+import json as _json
+import papermill as _pm
+from google.protobuf import text_format as _text_format, json_format as _json_format
+import importlib as _importlib
+import datetime as _datetime
+from pyspark import SparkConf, SparkContext
+import sys as _sys
+import six as _six
+from flytekit import __version__
+from flytekit.bin import spark_executor
+from flytekit.sdk.types import Types as _Types
+from flytekit.common.types import helpers as _type_helpers, primitives as _p
```

Comment:
can you make _p longer?  like _primitive_types?

Reply:
done